packages/vega-geo/src/util/contours.js

```diff
@@ -62,8 +62,8 @@ export default function() {
   // Marching squares with isolines stitched into rings.
   // Based on https://github.com/topojson/topojson-client/blob/v3.0.0/src/stitch.js
   function isorings(values, value, callback) {
-    var fragmentByStart = new Array,
-        fragmentByEnd = new Array,
+    var fragmentByStart = [],
```

Comment:
Note that this was the only violator of the "don't use `new Array` lint, and it didn't feel worth it to do a follow up just for that.